src/wasm/compat/mount.rs

```diff
@@ -850,6 +851,47 @@ impl WasmMount {
         Ok(())
     }
 
+    /// Share a file snapshot
+    #[wasm_bindgen(js_name = shareFile)]
+    pub async fn share_file(&mut self, path_segments: Array) -> TombResult<String> {
```

Comment:
should this just return a `WasmSharedFile`? I'm not sure if its better to just spit out the b64 data right here or somewhere else, like leaving that to the frontend